### DISCUSSION TITLE
[BUG] 设置图可使用的内存

### DISCUSSION THREAD

**Discussion by xglreal** [alibaba/GraphScope#2474] [Q&A] [answered]
载入9G的亿级关系数据，报错显示4G内存不够分配。哪里可以设置该图可使用的内存大小？

**Comment by welcome** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! And a maintainer will get back to you shortly!
Please feel free to contact us on [DingTalk](https://h5.dingtalk.com/circle/healthCheckin.html?dtaction=os&corpId=dingb4b76e0fa9ccd8d516fd0f0acdf19f55&68e71190=0fe34ec0&cbdbhh=qwertyuiop), WeChat account(graphscope) or [Slack](http://slack.graphscope.io/). We are happy to answer your questions responsively.

**Comment by sighingnow** [comment, marked as answer]
See the 9th item in https://graphscope.io/docs/frequently_asked_questions.html

> What’s the relationship between k8s_vineyard_mem, vineyard_shared_mem and k8s_engine_mem?

**Comment by sighingnow** [comment]
Moving to discussion.
